Magi/Aine filia Saoirse

```diff
@@ -0,0 +1,87 @@
+# Aine filia Saoirse
+
+Aine filia Saoirse, is a celtic magi who traveled to the Netherlands to join Foedus Caligae Aqueae a newly formed covenant. Her parens, Saoirse Ignislucida, is her aunt. Saoirse thinks herself as a good magi and thinks a strong Gift could be a family trait. Instead she finds a child with subtle magic instead (Gentle gift). This is how Aine got kiddnaped and got taught magic. Aine doenst know the actual familiar relationship she has with Saoirse. Regardless they have a strong realtionship and Aine trusts her parens with issues he has and asks for advice. Aine wants to become a hoplit, but get distracted by new spells and her covenant mates.  
+
+Aine started as an ignem spesialist, who branshed out to Coprus and Vim. She has a good overall spread of scores in magical arts.   
+
+**Characteristics:** Int +2, Per +1, Pre +0, Com +1, Str -2, Sta +3, Dex +1, Qik +1  
+**Size:** 0  
+**Age:** 35  
+**Decrepitude:** 0 (0)  
+**Warping Score:** 0 (1)  
+**Confidence Score:** 1 (6)
+
+## Virtues and Flaws:
+
+- The Gift
+- Gentle Gift
+- Puissant Perdo
+- Puissant Creo
+- Affinity with Ignem
+- Affinity with Intellego
+- Cyclic magic - Day
+- Booklearner
+- Mastered spells
+- Enduring Constitution 
+
+- Supernatural Nuisance
+- Deficient form - Herbam 
+- Overconfident
+- Busybody
+- Warped magic
+- Weak enchanter
+
+
+**Personality Trait:** Overconfident +3, Busybody +1, Sociable with mundanes +2  
+**Reputations:** House Acclaim +2, Generous Spender +1, 
+
+**Combat:**  
+*Dodge*: Init -1, Attack +0, Defense -1, Damage +0                                                                                                    
+
+**Soak:** +3  
+**Fatigue Leves:** OK, 0, 0, -2, -4, Unconscious  
+**Wound Penalties:** -1 (1-5), -3 (6-10), -5 (11-15), Incapacitated (16-20), Dead (21+) , Decrease total penalty of wounds by -1 (Enduring constitution)
+
+## Abilities:
+
++ Area Lore: Celtic 1 (myth and magic)
++ Artes Liberales 1 (ritual magic)
++ Athletics 1 (Dodge)
++ Awareness 2 (Alertness)
++ Concentration 1 (Labwork)
++ Finesse 1 (Ignem)
++ Folk Ken 2 (oppocite sex)
++ Guile 1 (fast talk)
++ Infernal Lore +1 (Demons)
++ Language (native): Celtic 5 (Storytelling)
++ Language: Latin 4 (Academic use)
++ Language: Middle Dutch 4 (Arnhem Dialect)
++ Magic Lore 1 
++ Magic Theory +5 (Creo)
++ Parma Magica +2 (Ignem)
++ Penetration +3 (Ignem)
++ Single Weapons +2 (Short Sword)
++ Stealth +1 (Shadow)
```

Comment:
No need for '+' in front of the level.

Also I've tended to add the specialty before the level